Rakefile

```diff
@@ -69,4 +69,23 @@ begin
 
 rescue LoadError
 end
+
+desc 'Add Alma and Rocky to metadata.json'
+task :extend_metadata do
+  require 'json'
+  metadata = JSON.load_file('metadata.json')
+  if metadata['operatingsystem_support'].any?{|operatingsystem| operatingsystem['operatingsystem'] == 'CentOS' }
+    #versions = metadata['operatingsystem_support'].select{ |os| os['operatingsystem'] == 'CentOS'}['operatingsystemrelease']
+    versions = metadata['operatingsystem_support'].select{ |os| os['operatingsystem'] == 'CentOS'}.last['operatingsystemrelease']
+    alma = {'operatingsystem' => 'AlmaLinux', 'operatingsystemrelease' => versions}
+    rocky = {'operatingsystem' => 'Rocky', 'operatingsystemrelease' => versions}
+    metadata['operatingsystem_support'] << alma unless metadata['operatingsystem_support'].any?{|operatingsystem| operatingsystem['operatingsystem'] == 'AlmaLinux' }
+    metadata['operatingsystem_support'] << rocky unless metadata['operatingsystem_support'].any?{|operatingsystem| operatingsystem['operatingsystem'] == 'Rocky' }
+    puts metadata
+    File.open('metadata.json', 'w') do |f|
+      f.write(JSON.pretty_generate(metadata) + "\n")
+    end
+  end
+end
+
```

Comment:
Is this supposed to be in this change? Seems unrelated to docs cleanup.

Reply:
ah this wasn't supposed to be in the PR :D will clean it up